inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -397,50 +398,71 @@ def _curators_note2marc(self, key, value):
 
 
 @hepnames.over('experiments', '^693..')
-@utils.for_each_value
-@utils.filter_values
-def experiments(self, key, value):
+def experiments(self, key, values):
     """Information about experiments.
 
     Currently the ``id`` subfield stores the name of the experiment. It should
     be moved to storing id of the experiment as soon as experiments are
     available as records.
     """
```

Comment:
Is this docstring still relevant? I mean, `693__i` is basically an unused field: https://inspirehep.net/search?ln=en&cc=HepNames&of=text&ot=693__i&p=693__i:**.